app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/SessionDataExtractor.scala

```diff
@@ -56,30 +56,35 @@ trait SessionDataExtractor extends Results {
         Future.successful(Redirect(baseRoutes.StartController.start()))
     }
 
-  def getNumberOfClaims(draftClaim: DraftClaim): Option[SelectNumberOfClaimsType] =
+  def getNumberOfClaims(draftClaim: DraftClaim): SelectNumberOfClaimsType =
     draftClaim
       .fold(identity)
       .selectNumberOfClaimsAnswer
       .flatMap(
         _.fold(_.selectNumberOfClaimsChoice, a => Option(a.selectNumberOfClaimsChoice))
       )
+      .getOrElse(
+        SelectNumberOfClaimsType.Individual
+      ) //If the bulk claim is disabled, the user never sees the Select Number of Claims page
```

Comment:
Was just chatting with Mikail and lack of specific unit test for this getOrElse came up. We need to get into the habit of unit testing the heck out of generic components like this. Can you add one for this.

Reply:
There is a test for this. Just added with this pull request

Reply:
Sorry. I should have been more specific. I mean a SessionDataExtractorSpec test